crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/presets.rs

```diff
@@ -17,3 +20,10 @@ pub fn get_utilities_preset(preset: &UseSortedClassesPreset) -> UtilitiesConfig
         UseSortedClassesPreset::TailwindCSS => TAILWIND_LAYERS.as_slice(),
     }
 }
+
+pub fn get_variants_preset(preset: &UseSortedClassesPreset) -> VariantsConfig {
```

Comment:
Wonder if we can unify into a "preset" struct or something for simplicity, instead of two separate things as currently.

Reply:
Yeah probably we can lift the split on kind of preset, instead of this, I will give it a go